content/doc/administration/requirements/jenkins-on-java-11.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: documentation
+title: Running Jenkins on Java 11
+---
+
+If you need to upgrade the JVM version used to run Jenkins from Java 8 to Java 11, please link:/doc/administration/requirements/upgrade-java-guidelines[follow this guidelines].
+
+NOTE: At this moment, it is not required to upgrade the JVM version in order to run a Jenkins core more recent than `2.155`.
+
+Since Jenkins `2.155`, it is possible to run Jenkins with Java 11.
+However, it was greatly simplified since version `2.163`. 
```

Comment:
I removed it in https://github.com/jenkins-infra/jenkins.io/pull/2097/commits/988642e89d9c9f59674dba9de7d24ebe9dc4e827

Reply:
What I meant is that I could think I can run Jenkins 2.163 without params, but actually, it needs `--enable-future-java`. It's not said that running with the `java` command only works as of 2.164. We should say what you need to run in 2.163 or just assume, that the first java11-full-supported version is the 2.164, well, perhaps _java11-easy-supported_ version :wink:

Reply:
I stated [here](https://github.com/jenkins-infra/jenkins.io/pull/2097/files#diff-8885f246113bf709b2ad52efd631a80cR13) that it's from `2.155` but easier since `2.163`. I might update the all documentation to say `2.164`. And in general, this documentation will need to be updated to use LTS version once we have LTS GA.

Reply:
+1. IMO, only talk about 2.164+, and use `footnote:[to provide the full story]`

Reply:
footnote provided in https://github.com/jenkins-infra/jenkins.io/pull/2097#discussion-diff-254974278R13